knexfile.js

```diff
@@ -1,7 +1,51 @@
 const config = {
-  client: 'pg',
-  connection: process.env.DATABASE_URL || `postgres://${process.env.USER}@127.0.0.1:5432/livejazz`,
-  searchPath: 'public',
+  development: {
+    client: 'pg',
```

Comment:
This knex file makes me sad because it violates the "config" factor:

https://12factor.net/config

```In a twelve-factor app, env vars are granular controls, each fully orthogonal to other env vars. They are never grouped together as “environments”, but instead are independently managed for each deploy. This is a model that scales up smoothly as the app naturally expands into more deploys over its lifetime.

```